rs/sns/governance/src/governance.rs

```diff
@@ -8596,7 +8660,7 @@ mod tests {
         // After we run our periodic tasks, the version should be marked as successful
         governance.run_periodic_tasks().now_or_never();
 
-        assert!(governance.proto.pending_version.is_none());
+        assert_eq!(governance.proto.pending_version, None);
```

Comment:
:D glad that you're doing this, it's much easier to debug

Reply:
+1